src/transport/SecureSessionTable.h

```diff
@@ -60,9 +60,9 @@ class SecureSessionTable
      *          has been reached (with CHIP_ERROR_NO_MEMORY).
      */
     CHECK_RETURN_VALUE
-    Optional<SessionHandle> CreateNewSecureSession(SecureSession::Type secureSessionType, uint16_t localSessionId,
-                                                   NodeId peerNodeId, CATValues peerCATs, uint16_t peerSessionId,
-                                                   FabricIndex fabric, const ReliableMessageProtocolConfig & config)
+    Optional<SessionHandle> CreateNewSecureSessionForTest(SecureSession::Type secureSessionType, uint16_t localSessionId,
```

Comment:
I wonder if, as a followup, we can condition this on some "unit testing enabled" define....